tfjs-layers/src/engine/container_test.ts

```diff
@@ -385,9 +385,9 @@ describeMathCPUAndGPU('Container', () => {
 
   it('getLayer works by index', () => {
     const [container, layers] = createSimpleTwoLayerContainer();
-    expect(container.getLayer(null, 0)).toEqual(layers[0]);
-    expect(container.getLayer(null, 1)).toEqual(layers[1]);
-    expect(container.getLayer(null, 2)).toEqual(layers[2]);
+    expect(container.getLayer(0)).toEqual(layers[0]);
+    expect(container.getLayer(1)).toEqual(layers[1]);
+    expect(container.getLayer(2)).toEqual(layers[2]);
```

Comment:
In addition to this new API, we should still test the `(null, index)` case since it's part of our public api.

Reply:
Thanks! Done.